### PR TITLE
static error pages

### PR DESCRIPTION
**Pull request by lammia07**

### CODE REVIEW

File: features/errors/404.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css"
+        integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="/css/styles.css">
+    <!-- favicon -->
+    <link rel="icon" type="image/vnd.microsoft.icon" href="/img/favicon.ico">
+
+    <title>404</title>
```
Comment (m4rien0) [inline features/errors/404.html]:
```suggestion
    <title>FH Joanneum - 404</title>
```

File: features/errors/500.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css"
+        integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="/css/styles.css">
+    <!-- favicon -->
+    <link rel="icon" type="image/vnd.microsoft.icon" href="/img/favicon.ico">
+
+    <title>404</title>
```
Comment (m4rien0) [inline features/errors/500.html]:
```suggestion
    <title>FH Joanneum - 500</title>
```

File: features/router.js [hunk 3]
```diff
@@ -1,9 +1,18 @@
...
+    res.sendFile(path.join(__dirname, "/errors/404.html"));
+});
+
+router.get("/500", (req, res) => {
+    res.sendFile(path.join(__dirname, "/errors/500.html"));
+});
+
 router.get("/", (req, res) => {
     res.redirect("/projects");
 });
```
Comment (m4rien0) [inline features/router.js]: You can change the single quotes setting in the .prettierrc to true and let the double quotes be replaced by single ones (on this file)

**Review by m4rien0 — CHANGES_REQUESTED** [review]

File: features/router.js (hunk 3 above)
Comment (lammia07) [reply features/router.js]: merge with master was done, now prettier changes to single quote

**Review by m4rien0 — APPROVED** [review]

**Comment by m4rien0** [comment]
Added static error pages to the application.

**Pull request merged**